includes/polling/ports.inc.php

```diff
@@ -952,5 +979,9 @@
     unset($this_port, $port);
 } //end port update
 
+// Update the poll_time, poll_prev and poll_period of all ports in an unique request
+$updated = dbUpdate($device_global_ports, 'ports', '`port_id` IN ('.implode(',', $port_ids).')', array($device['device_id'], $device_global_ports['poll_prev']));
```

Comment:
dbUpdate doesn't link IN being used like that. It will escape it in a way that can fail.  There are no placeholders, yet you specify two values.
You could just use this: `DB::table('ports')->whereIn('port_id', $port_ids)->update($device_global_ports);`